lightcycle-api/src/main/java/com/soundcloud/lightcycle/FragmentLightCycle.java

```diff
@@ -1,13 +1,18 @@
 package com.soundcloud.lightcycle;
 
+import android.annotation.TargetApi;
 import android.app.Activity;
 import android.app.Fragment;
+import android.content.Context;
+import android.os.Build;
 import android.os.Bundle;
 import android.view.MenuItem;
 import android.view.View;
 
+@TargetApi(Build.VERSION_CODES.HONEYCOMB)
 public interface FragmentLightCycle<T extends Fragment> {
     void onAttach(T fragment, Activity activity);
+    @TargetApi(23) void onAttach(T fragment, Context context);
```

Comment:
Could you break the line after `@TargetApi(23) `, please ? 

Not your fault, we did not setup a proper code formatter and checker. 
https://github.com/soundcloud/lightcycle/issues/65